homeassistant/components/linkplay/strings.json

```
@@ -43,7 +43,13 @@
     },
     "select": {
       "audio_output_hardware_mode": {
-        "name": "Audio output hardware mode"
+        "name": "Audio output hardware mode",
+        "state": {
+          "optical": "Optical",
+          "line_out": "Line Out",
```

"[line out](https://en.wiktionary.org/wiki/line_out)" needs sentence-casing:

```suggestion
          "line_out": "Line out",
```